pkg/oss/nodeserver.go

```diff
@@ -152,6 +154,12 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 			opt.AuthType = strings.ToLower(strings.TrimSpace(value))
 		} else if key == "rolename" {
 			opt.RoleName = strings.TrimSpace(value)
+		} else if key == "rolearn" {
+			opt.RoleArn = strings.TrimSpace(value)
+		} else if key == "providerarn" {
+			opt.RoleArn = strings.TrimSpace(value)
```

Comment:
typo? why providerarn is assigned to RoleArn?

Reply:
oh, sorry that's a mistake I made in merging, oidcProviderArn is used acutally.